taccsite_cms/templates/base.html

```diff
@@ -46,14 +46,14 @@
   {% render_block "css" %}
 </head>
 
-<body class="o-site  {% block page_type_class %}{% endblock page_type_class %}">
+<body class="{% block page_type_class %}{% endblock page_type_class %}">
```

Comment:
Removal of o-site (and child classes) is allowed because I found [a simpler way to make a sticky footer](https://github.com/TACC/tup-ui/pull/18/files#diff-7a0f61a98c72dc6bd12575fbe90c75fa33d020500b696faccf41904e63bfad2fR65-R70).